aten/src/ATen/native/native_functions.yaml

```diff
@@ -1772,8 +1774,7 @@
     SparseCPU: hspmm_sparse_cpu
     SparseCUDA: hspmm_sparse_cuda
 
-# This "raw copy" doesn't handle conversions NOR does it handle non-blocking.
-- func: raw_copy_sparse_(Tensor self, Tensor src) -> Tensor
+- func: raw_copy_sparse_(Tensor self, Tensor src, bool non_blocking=false) -> Tensor
```

Comment:
OK, so what about conversions? If I copy from FloatTensor to DoubleTensor a conversion happens. What happens here?

Reply:
Also, if all of this is fixed, let's call it something besides `raw_copy_sparse`, like `copy_sparse_to_sparse`.

Reply:
So what is the expected output after type conversions here? For instance, in dense tensor, when copy a FloatTensor to DoubleTensor, all values in FloatTensor will cast Float to Double. 
```
x = torch.tensor([1., 1.], dtype=torch.float32)
y = torch.tensor([2., 2.], dtype=torch.float64)
y.copy_(x)
print(y, y.dtype)
---------------
(tensor([1., 1.], dtype=torch.float64), torch.float64)
```
And this also holds for sparse tensor. `raw_copy_sparse_ ` should have already served this purpose I guess?

Reply:
Yes, so maybe this already handles conversions now that it was changed from `clone()` to `copy()`. In that case, nothing more to do but change the function name :)

Reply:
Right, now sparse copy_() does the type conversion by using [dense copy() for indices and values](https://github.com/weiyangfb/pytorch/blob/83e626d53e4340807e4649de2e3e06a0b0dfaf92/aten/src/ATen/native/sparse/SparseUtils.h#L57-L59)